create_pdfs.py

```diff
@@ -37,8 +37,9 @@ def parse_args(args=sys.argv[1:]):
     parser.add_argument('-i', '--issued-date', help='optional issue date')
     parser.add_argument(
         '-T',
```

Comment:
This is unnecessary change that would break my and possible other's development cycle.  I have scripts to auto gen many scripts that use the 'T' arg.  We shouldn't change them on a whim.